.github/workflows/labeler.yml

```
@@ -0,0 +1,14 @@
+name: "Labeler"
+on:
+  pull_request:
+    types: [opened, edited]
```

These are all the events I found to use to check PRs
```suggestion
    types:
      - opened
      - reopened
      - edited
      - synchronize
```

---

The labeling is currently triggered by a match with the corresponding PR task list item when it's checked/unchecked. Curious when would `reopened` and `synchronize` be useful?

---

I figured we want to be safe. Someone could close a PR (or it gets closed from a bad rebase, etc) and then checks the box and then re-opens it; without `reopened`, do you know if `edited` is triggered for closed pull requests?

I believe having synchronize allows us to update this workflow in `master` and it will be run for other PRs. We can probably remove it since I don't think that's a big deal.

---

Thanks for the justifications. Was planning to add them in anyway as we aren't billed for GHAs, was just curious.